cmd/network/governance/show.go

```diff
@@ -254,8 +255,15 @@ var govShowCmd = &cobra.Command{
 			}
 		}
 
+		// Display the proposal content.
+
+		fmt.Println("=== PROPOSAL CONTENT ===")
+		proposal.Content.PrettyPrint(ctx, " ", os.Stdout)
```

Comment:
It's a toplevel print, so we don't need any indent.

```suggestion
		proposal.Content.PrettyPrint(ctx, "", os.Stdout)
```